applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_base_solver.py

```diff
@@ -579,6 +596,10 @@ def _create_solution_scheme(self):
 
 
     def _create_convection_diffusion_solution_strategy(self):
+        time_integration_method = self.settings["time_integration_method"].GetString()
+        if time_integration_method == "explicit":
+            convection_diffusion_solution_strategy = self._create_runge_kutta_4_strategy()
+            return convection_diffusion_solution_strategy
```

Comment:
Why are you adding this to the base solver if you are creating your own explicit one?

Reply:
Moving to explicit solver.